### DISCUSSION TITLE
Having some big problems now (1.0.0.7)

### DISCUSSION THREAD

**Discussion by adamesq** [rrirower/Channel-Feed-Manager#102] [Q&A] [answered]
1) I opened the feed created in 1.0.0.6 in 1.0.0.7, then verified no errors with the validate contents tool. Saved the feed.
2) Copied the feed and opened in the JSON Schema Validator. I got the same error: Required properties are missing from object: version, assets. (SAME error as in 1.0.0.6)
HERE's where it really went worse...
3) I deleted the feed in CFM, thinking the problem was just that I needed to rebuild in 1.0.0.7. 
4) Tried importing the working JSON feed in CFM using "Add to your feed library"
5) Opens in CFM EMPTY - no contents from the feed created in 1.0.0.6 came in.
6) Tried adding categories and just the main series info (with the same ID from the feed), now I get ATTENTION! There were one or more errors while attempting to load your feed file items. Some items may be missing. Please review the log file for more info. Log file shows the same message! 😱

UGHHH...at this point, just want to get the feed file library created in 1.0.0.6 back, but it seems CFM doesn't put it in the bin...just gone.

Is CFM unable to create a Feed file library from a feed it created?

**Comment by adamesq** [comment]
Following up! 
JUST got a feed imported back into CFM from a few hours ago. Just missing a few entries from today. (de-stressing!)

SO, resaved from 1.0.0.7. Remaining issue now is just that I'm still getting "Required properties are missing from object: version, assets." in JSON Schema Validator. Same issue as 1.0.0.6. I don't want to risk screwing up my feed library again! 😅 Holding for suggestions!

**Reply by rrirower** [reply, marked as answer]
First, I wouldn't bother with the json schema validator. It only looks for syntax errors. In fact, I've never run it. I use the validation in CFM.

Second, if you've deleted a feed file outside of CFM, you created an orphan in its library. You should be using the right mouse click options on the first form.

**Reply by rrirower** [reply]
Can I suggest if you haven't already, to read the Wiki for CFM.  It seems like you might be shooting yourself in the foot because you're in a rush.

**Comment by adamesq** [comment]
> Can I suggest if you haven't already, to read the Wiki for CFM. It seems like you might be shooting yourself in the foot because you're in a rush.
Yeahhh...I jumped a little quickly on this one. It's been quite a learning detour this week from Amazon/EC2 to SDK, but no excuses...
GUILTY as charged.
![os7VZyT](https://github.com/rrirower/Channel-Feed-Manager/assets/34196956/9d14ff88-f520-4234-8df2-93db6bb594e2)
